yt/visualization/plot_modifications.py

```diff
@@ -1632,6 +1633,7 @@ def __init__(self, width, p_size=1.0, col='k', marker='o', stride=1,
         self.ptype = ptype
         self.minimum_mass = minimum_mass
```

Comment:
So I guess we're still taking `minimum_mass` but you'd like to deprecate it in favor of using a particle filter? that seems ok to me although we need to bear in mind that we want old scripts to keep on working and producing the same result, at least until we make a major version number change to signal that we are breaking backward compatibility in places.

That's ok, we can still deprecate `minimum_mass` but to do that it will still need to keep on working the same way until the functionality is completely removed. So, if someone does pass in a `minimum_mass` not set to `None` here you'd issue a deprecation warning using `issue_deprecation_warning` in `yt.funcs` with a suggestion to use a particle filter.